system/services/assetManager/ImageManipulationService.cfc

```diff
@@ -38,15 +38,17 @@ component displayname="Image Manipulation Service" {
 		,          string  cropHint            = ""
 		,          string  useCropHint         = false
 		,          struct  fileProperties      = {}
+		,          string  ignoreDimension     = false
 	) {
 		var args = arguments;
 
 		if ( arguments.useCropHint && arguments.cropHint.len() ) {
 			args.cropHintArea = _getCropHintArea(
-				  image    = args.asset
-				, width    = args.width
-				, height   = args.height
-				, cropHint = args.cropHint
+				  image           = args.asset
+				, width           = args.width
+				, height          = args.height
+				, cropHint        = args.cropHint
+				, ignoreDimension = $helpers.isTrue( args.ignoreDimension )
```

Comment:
As above, if the argument is a `boolean` type, then no need for the `isTrue()` helper.